### PR TITLE
(NOBIDS) rename api error func

### PR DESCRIPTION
**Pull request by LuccaBitfly**
Renames two functions, cmon just approve, trust me

### <samp>🤖 Generated by Copilot at 77cf0be</samp>

No summary available (Limit exceeded: required to process 53072 tokens, but only 50000 are allowed per call)

### CODE REVIEW

File: handlers/api.go [hunk 1]
```diff
@@ -4095,12 +4095,12 @@ func adjustQueryResults(data []interface{}, adjustQueryEntriesFuncs ...func(map[
 	return nil
 }
 
-// SendErrorResponse exposes sendErrorResponse
```
Comment (D13ce) [inline handlers/api.go:4098]: **Minor:** This comment now is outdated. Also, search for `sendErrorResponse` within the sources and you will find two more locations. Those are both code that is commented out but I suggest to either remove it completely or to fix it.

File: handlers/api.go [hunk 2]
```diff
@@ -4095,12 +4095,12 @@ func adjustQueryResults(data []interface{}, adjustQueryEntriesFuncs ...func(map[
...
-func SendErrorResponse(w http.ResponseWriter, route, message string) {
-	sendErrorResponse(w, route, message)
+// SendBadRequestResponse exposes sendErrorResponse
+func SendBadRequestResponse(w http.ResponseWriter, route, message string) {
+	sendBadRequestResponse(w, route, message)
 }
 
-func sendErrorResponse(w http.ResponseWriter, route, message string) {
+func sendBadRequestResponse(w http.ResponseWriter, route, message string) {
 	sendErrorWithCodeResponse(w, route, message, 400)
```
Comment (D13ce) [inline handlers/api.go:4104]: **Nitpick:** I know this does not come from your PR but as you have touched this function I suggest to replace the hardcoded 400 with `http.StatusBadRequest`.

Related: https://github.com/gobitfly/eth2-beaconchain-explorer/blob/77cf0bef9cdc9d872321f989479eca66e4ce334c/handlers/api.go#L4107-L4109
and `http.StatusInternalServerError`.

File: handlers/api.go [hunk 3]
```diff
@@ -4095,12 +4095,12 @@ func adjustQueryResults(data []interface{}, adjustQueryEntriesFuncs ...func(map[
 	return nil
 }
 
-// SendErrorResponse exposes sendErrorResponse
-func SendErrorResponse(w http.ResponseWriter, route, message string) {
-	sendErrorResponse(w, route, message)
+// SendBadRequestResponse exposes sendErrorResponse
+func SendBadRequestResponse(w http.ResponseWriter, route, message string) {
+	sendBadRequestResponse(w, route, message)
 }
```
Comment (D13ce) [inline handlers/api.go:4101]: **Question:** Why not remove `sendBadRequestResponse` and only have `SendBadRequestResponse`? I know, it was like this before, but I am asking as you have touched it.
IF you want to change it, you should also take a look at https://github.com/gobitfly/eth2-beaconchain-explorer/blob/77cf0bef9cdc9d872321f989479eca66e4ce334c/handlers/api.go#L4123-L4126 for consistency.

**Review by D13ce — COMMENTED** [review]
ptal

File: handlers/api.go (hunk 1 above)
Comment (LuccaBitfly) [reply handlers/api.go:4098]: remove

File: handlers/api.go (hunk 2 above)
Comment (LuccaBitfly) [reply handlers/api.go]: fix

File: handlers/api.go (hunk 3 above)
Comment (LuccaBitfly) [reply handlers/api.go]: fix

**Review by D13ce — APPROVED** [review]
looks lgtm to me

**Pull request merged**